### PR TITLE
Jbide 24690 prepare for final

### PR DESCRIPTION
**Pull request by apupier**
https://github.com/jbosstools/jbosstools-fuse-extras/pull/68 need to be merged before this one

### CODE REVIEW

File: jboss-fuse-sap-tool-suite/targetplatform/pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 		<version>10.0.0-SNAPSHOT</version>
 	</parent>
 
-	<artifactId>org.fusesource.ide.sap.targetplatform</artifactId>
+	<artifactId>sap-tooling-multiple</artifactId>
```
Comment (bfitzpat) [inline jboss-fuse-sap-tool-suite/targetplatform/pom.xml:13]: Is changing this going to break anything down the line?

**Review by bfitzpat — APPROVED** [review]
I suspect this is fine - but am always concerned when we start changing artifact IDs. I'm not sure I'm qualified to review it however. :) 

@punkhorn ? @nickboldt ?

File: jboss-fuse-sap-tool-suite/targetplatform/pom.xml (hunk 1 above)
Comment (apupier) [reply jboss-fuse-sap-tool-suite/targetplatform/pom.xml:13]: I don't think that it is used anywhere else, I just tried to have coherent name with what has been changed in Fuse Tooling too.

Comment (bfitzpat) [reply jboss-fuse-sap-tool-suite/targetplatform/pom.xml:13]: Cool. Then it's probably fine. :)

**Pull request merged**